### PR TITLE
Automated chart bump defaultstorageclass-0.0.6

### PR DESCRIPTION
**Pull request by d2iq-dispatch**
Add release note or "none":
```release-note
- update client-go to 0.19.2 to support k8s 1.16-1.21
- use the distroless image and run as nonroot user to address image CVEs
```

### CODE REVIEW

**Comment by joejulian** [comment]
@mesosphere-mergebot backport release/3.2

**Comment by joejulian** [comment]
rats

**Review by armandgrillet — APPROVED** [review]

**Comment by joejulian** [comment]
@mesosphere-mergebot backport release/3.2

**Comment by joejulian** [comment]
@mesosphere-mergebot backport release/3.2

**Comment by joejulian** [comment]
@mesosphere-mergebot backport release/3.2

**Comment by joejulian** [comment]
@mesosphere-mergebot backport release/3.2

**Review by joejulian — APPROVED** [review]

File: addons/defaultstorageclass-protection/defaultstorageclass-protection.yaml [hunk 1]
```diff
@@ -1,12 +1,11 @@
----
 apiVersion: kubeaddons.mesosphere.io/v1beta1
 kind: ClusterAddon
 metadata:
   name: defaultstorageclass-protection
   labels:
     kubeaddons.mesosphere.io/name: defaultstorageclass-protection
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "0.0.5-1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.0.5-2"
```
Comment (samvantran) [inline addons/defaultstorageclass-protection/defaultstorageclass-protection.yaml:8]: Shouldn't this be `0.0.6-1`?

File: addons/defaultstorageclass-protection/defaultstorageclass-protection.yaml [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
     appversion.kubeaddons.mesosphere.io/defaultstorageclass-protection: "0.0.5"
```
Comment (samvantran) [inline addons/defaultstorageclass-protection/defaultstorageclass-protection.yaml:9]: Shouldn't this be `0.0.6`?

**Comment by joejulian** [comment]
@mesosphere-mergebot backport release/3.3

**Pull request merged**